types/react-loadable/test/index.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Loadable = require('react-loadable');
+import * as Loadable from 'react-loadable';
```

Comment:
`react-loadable` exports itself as a commonjs export so it should be imported with `import Loadable = require('react-loadable')`

Reply:
why?

I mean - do you have any particular reason why the import/export can not be used with this module?